src/data/faq.json

```diff
@@ -2171,66 +2171,299 @@
+                    "term": "Science-Blog",
+                    "anchor": "science_blog",
+                    "description": "The Corona-Warn-App Science Blog covers various scientific topics related to the Corona-Warn-App, such as its effectiveness, benefits, as well as results of scientific evaluations conducted by the Robert Koch Institute. See <a href='../science'>Science-Blog</a>."
+                },
+                {
+                    "term": "Short Lived Random Bluetooth ID",
+                    "anchor": "short_lived_random_bluetooth_id",
+                    "description": "Unique key generated several times an hour, derived from the random device key and exchanged via Bluetooth between smartphones that are nearby and have a tracing app such as the Corona-Warn-App on them."
+                },
                 {
                     "term": "Signature",
                     "anchor": "signature",
-                    "description": "A digital signature is a signature that is created by a cryptographic function from a digital document. Digital signatures provide the following properties: <ul><li>Non-repudiation: The issuer cannot retrospectively deny signing the document.</li><li>Integrity: Altering the document invalidates the signature.</li><li>Authenticity: It is clear who signed the document.</li></ul> <br/><br/> As a consequence, the signature must fulfill the following requirements: <br/><ul><li>The signature must be computationally verifiable </li><li>Verifying the signature must be easy - forging a valid signature must be hard</li><li>The signature is only valid for the signed document and no other documents </li><li>The issuer must not be able to deny signing the document.</li></ul>"
+                    "description": "A digital signature is a signature that is created by a cryptographic function from a digital document. Digital signatures provide the following properties: <ul><li>Non-repudiation: The issuer cannot retrospectively deny signing the document.</li><li>Integrity: Altering the document invalidates the signature.</li><li>Authenticity: It is clear who signed the document.</li></ul> As a consequence, the signature must fulfill the following requirements: <ul><li>The signature must be computationally verifiable </li><li>Verifying the signature must be easy - forging a valid signature must be hard</li><li>The signature is only valid for the signed document and no other documents </li><li>The issuer must not be able to deny signing the document.</li></ul>"
+                },
+                {
+                    "term": "Standardized Name",
+                    "anchor": "standardized_name",
+                    "description": "Standardized names refer to names that are additionally presented with capital letters in standardized Latin script due to their country-specific spelling. This can occur, for example, with Cyrillic, Arabic, but also German script. For example, a \"Ö\" becomes a \"OE\"."

```

Comment:
The examples are both vowels, so "a" should be "an". It's better though just to remove the "a".

For example, a \"Ö\" becomes a \"OE\".

=>

For example \"Ö\" becomes \"OE\".

Reply:
Will be solved in the next commit, thanks!